rotkehlchen/db/upgrades/v23_v24.py

```diff
@@ -0,0 +1,37 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def upgrade_v23_to_v24(db: 'DBHandler') -> None:
+    """Upgrades the DB from v23 to v24
+
+    - Deletes the AdEx used query ranges.
+    - Drops the existing AdEx events table.
+    - Creates the new AdEx events table that includes `log_index` in the PK.
+    """
+    cursor = db.conn.cursor()
+    # Delete AdEx used query ranges, drop the events table and re-create it
+    cursor.execute('DELETE FROM used_query_ranges WHERE name = "adex_events";')
```

Comment:
This is wrong and will lead to the very nice bug where the data is deleted but the used query ranges are still there. 

https://github.com/rotki/rotki/blob/d8e74546104b6297c42958cf2f1a8232f27f9d59/rotkehlchen/db/dbhandler.py#L939-L947

As you can see you need to do a `LIKE` and not equality, since name is `${prefix}_${address}`. Which is something especially you should know since you wrote this code man...

Reply:
True this is a mistake, sorry. I overlooked that for defi modules we append the user address. I will fix it